.eslintrc

```diff
@@ -699,21 +700,24 @@ rules:
     - 2
     - 3
 
-  # Enforce a maximum number of parameters in function definitions.
+  # Limits number of parameters that can be used in the function declaration.
   max-params:
     - 1
     - 5
 
+  # Enforce a maximum number of statements allowed per line.
+  max-statements-per-line:
+    - 1
+    -
+      max: 2
```

Comment:
Seems fine for now. If we see a rash of inappropriate two-statement lines popping up, we can reevaluate.